docs/content/010-getting-started/03-tutorial/04-chapter-3-adding-mutations-to-your-api.mdx

```diff
@@ -63,10 +63,8 @@ export interface Context {
   db: Db
 }
 
-export function createContext(): Context {
-  return {
-    db
-  }
+export const context = {
+  db
 }
 ```
 
```

Comment:
Should we also add a note that we can pass a function for the context, and why we would have to do that in some cases?

Reply:
Yeah that would be nice